src/structure/assignBonds.ts

```diff
@@ -108,8 +109,10 @@ function areConnected(atom1: AtomSpec, atom2: AtomSpec) {
     else if (distSquared < 0.5) return false;
     // maybe duplicate position.
     else if (distSquared > maxsq) return false;
+    /**  FIXME: altLoc no longer exists on AtomSpec
```

Comment:
What's the significance here?

Reply:
we don't actually use altLoc, this is a biomolecule thing. This function used to handle it since it was extracted out of 3dmol (it is not publicly accessible), I need to remove this bit of code

Reply:
Okay! I might suggest leaving a comment that altloc is not supported, instead of a FIXME

Reply:
I removed this comment, since this more confusing than helpful